ansible_builder/steps.py

```diff
@@ -43,14 +43,36 @@ def __init__(self):
 
 
 class GalaxyInstallSteps(Steps):
-    def __init__(self, requirements_naming):
-        """Assumes given requirements file name has been placed in the build context
+    def __init__(self, requirements_naming, galaxy_keyring, galaxy_ignore_signature_status_code, galaxy_required_valid_signature_count):
+        """Assumes given requirements file name and keyring has been placed in the build context.
+
+        :param str galaxy_keyring: GPG keyring file used by ansible-galaxy to opportunistically validate collection signatures.
+        :param str galaxy_required_valid_signature_count: Number of sigs (prepend + to disallow no sig) required for ansible-galaxy to accept collections.
+        :param str galaxy_ignore_signature_status_code: GPG Status code to ignore when validating galaxy collections.
         """
+
+        env = ""
+        install_opts = f"-r {requirements_naming} --collections-path {constants.base_collections_path}"
```

Comment:
We should probably quote these values in the generated cmdline string (eg, paths with spaces)